src/main/resources/static/home/home.controller.js

```diff
@@ -10,36 +10,39 @@
         var vm = this;
 
         vm.user = null;
-        vm.allUsers = [];
-        vm.deleteUser = deleteUser;
+        // vm.allUsers = [];
+        // vm.deleteUser = deleteUser;
 
         initController();
 
         function initController() {
             loadCurrentUser();
-            loadAllUsers();
+            // loadAllUsers();
         }
 
         function loadCurrentUser() {
-            UserService.GetByUsername($rootScope.globals.currentUser.username)
-                .then(function (user) {
-                    vm.user = user;
-                });
+            // UserService.GetByUsername($rootScope.globals.currentUser.username)
```

Comment:
Why comment out? Just remove them right? (It'll still be stored in source control if we want to get the lines back someday)